src/Models/CsvCatalogItem.js

```diff
@@ -239,14 +301,14 @@ CsvCatalogItem.prototype._showInCesium = function() {
             return when(featurePromise, function(results) {
                 if (defined(results)) {
```

Comment:
Should add `&& results.length > 0` here, because returning an empty array is allowed (it'll happen when a layer is pickable but there are no features until the picked position).